pkg/sharedcli/profileflag/profileflag.go

```diff
@@ -27,7 +27,7 @@ const (
 type Options struct {
 	// EnableProfile is the flag about whether to enable pprof profiling.
 	EnableProfile bool
-	// ProfilePort is the TCP address for pprof profiling.
+	// ProfilingBindAddress is the TCP address for pprof profiling.
```

Comment:
The default is ":6060", which means it can accept requests from any source address(I tried), so I think we need the change the flag's default value from:
```go
	fs.StringVar(&o.ProfilingBindAddress, "profiling-bind-address", ":6060", "The TCP address for serving profiling(e.g. 127.0.0.1:6060, :6060). This is only applicable if profiling is enabled.")
```
to:
```go
	fs.StringVar(&o.ProfilingBindAddress, "profiling-bind-address", "127.0.0.1:6060", "The TCP address for serving profiling(e.g. 127.0.0.1:6060, :6060). This is only applicable if profiling is enabled.")
```

Reply:
This may introduce a user-facing change, I suggest updating the relevant comments. Hi @yanggangtony, can you help update it?

Reply:
/cc @Poor12

Reply:
// ProfilingBindAddress is the TCP address for pprof profiling.
// Defaults to :6060 if unspecified.

Reply:
yes, thanks for review..
rebased it..